### DISCUSSION TITLE
Tab navigation with nested navigation: show first tag screen on renavigate

### DISCUSSION THREAD

**Discussion by mateuy-dev** [adrielcafe/voyager#139] [Q&A]
Hi all,

I have an app with tab navigation, with nested navigation, but I'm having an issue.

**Events**
1. navigate inside a tab (there is a list and I go to a detail screen)
2. go to another tab
3. go to first tab

**Problem**
I expect to be show the "first" screen (the list), but the last navigated screen is shown (detail).

What I'm used to is that when a user clicks on a tab, the nested navigation should not be used, but start again with a new stack.

Is there somehow to achieve this?

Thanks,

**Comment by brucemax** [comment]
I got this effect by copying original TabNavigator and set disposeSteps = true for disposeBehavior

**Reply by jamesmbowler** [reply]
How? Can you show some code?
